contrib/chrusty/jsonschema/v1.3.9/buf.plugin.yaml

```diff
@@ -1,5 +1,6 @@
 version: v1
 name: buf.build/contrib/chrusty-jsonschema
 plugin_version: v1.3.9
+output_languages: [javascript]
```

Comment:
Not sure if we should categorize this as JavaScript. Maybe we need json api or open api or just to leave off language here (will fall into misc/other category?)

Reply:
Ye, I like that. Let's leave it out for now. `output_languages: []`